src/content/1.7/contribute/contribution-process/how-pull-requests-are-processed.md

```diff
@@ -85,6 +85,8 @@ After the Pull Request has finally passed the QA validation, it is merged in the
 
 ### Graphical summary
 
+(click on it to see full size)
```

Comment:
Could it be an alt attributes instead? https://dev.to/stephencweiss/markdown-image-titles-and-alt-text-5fi1

Reply:
I'm not sure to understand. Alt or Titles are there to help screen readers and SEO. How can they replace a statement to tell user he can click on image to see it full size ?

I added this statement because hugo reduces images so they are blurred. Blurry graphs are hard to read 😅

Reply:
Ohhh. You're right about alt and title but most of the time this is also used as helper. But you're right, this is maybe useless :sweat_smile: